gadget/install/params.go

```diff
@@ -36,4 +36,6 @@ type Options struct {
 type InstalledSystemSideData struct {
 	// KeysForRoles contains key sets for the relevant structure roles.
 	KeyForRole map[string]keys.EncryptionKey
+	// DeviceForRole maps a roles to their corresponding device nodes.
```

Comment:
this should probably clarify the point above about what we expect to be the device node here for an encrypted device